bundlewrap/items/actions.py

```diff
@@ -128,13 +131,23 @@ def get_result(self, *args, **kwargs):
         return status_code
 
     def run(self):
+        if self.attributes['data_stdin'] is not None:
+            data_stdin = self.attributes['data_stdin']
+            # Allow users to use either a string/unicode object or raw
+            # bytes.
+            if type(data_stdin) is not bytes:
+                data_stdin = data_stdin.encode('UTF-8')
```

Comment:
Should allow Faults as well. I think it would be best to just mirror `bundlewrap.utils.text.force_text`.